client/client.go

```diff
@@ -35,10 +35,14 @@ import (
 	"github.com/snapcore/snapd/dirs"
 )
 
-func unixDialer(_, _ string) (net.Conn, error) {
+func privateUnixDialer(_, _ string) (net.Conn, error) {
```

Comment:
One socket is available to all snaps (public), the other requires special access (private). `snapd` versus `snap` in this context seems a bit cryptic to me (especially considering the binary names), but I'll make the change.